libs/application/templates/financial-aid/src/fields/Confirmation/ApplicantConfirmation.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+
+import { ApproveOptions, FAFieldBaseProps } from '../../lib/types'
+import { confirmation } from '../../lib/messages'
+import { hasFiles, hasSpouse } from '../../lib/utils'
+import Confirmation from './Confirmation'
+
+const ApplicantConfirmation = ({ application }: FAFieldBaseProps) => {
+  const { answers, externalData } = application
+
+  const applicantHasSpouse = hasSpouse(answers, externalData)
+  const missingIncomeFiles =
+    answers.income === ApproveOptions.Yes && !hasFiles('incomeFiles', answers)
+
+  const firstStepText = () => {
+    switch (true) {
+      case (applicantHasSpouse && missingIncomeFiles):
+        return confirmation.nextSteps.contentBothMissingFiles
+      case (applicantHasSpouse):
+        return confirmation.nextSteps.contentSpouseMissingFiles
+      case (missingIncomeFiles):
+        return confirmation.nextSteps.contentMissingFiles
+      default:
+        return undefined
```

Comment:
is this necessary? if you take it doesn't it become undefined?

Reply:
It was giving me an error if I didn't have the default but I just tried to remove it and now it accepts it. Don't know what was happening 🤷‍♂️